client/dashboard/sites/overview/index.tsx

```diff
@@ -75,12 +123,21 @@ function SiteOverview() {
 					</VStack>
 					<OverviewCard title={ __( 'Plan' ) } icon={ wordpress } heading="TBA" />
 				</Grid>
-				<Divider orientation="horizontal" style={ { width: '100%', color: '#f0f0f0' } } />
-				<HStack className="site-overview-cards" spacing={ 6 } alignment="flex-start">
-					<VStack spacing={ 6 } justify="start">
+				<Divider
+					orientation="horizontal"
+					style={ { width: '100%', color: 'var(--wp-components-color-gray-100)' } }
```

Comment:
The width style isn't necessary here, it already goes full width.

I also think the name of the colour variable is wrong. Afaict the idea of the `<Divider>` is that it has no inherent colour, and so it is fully customisable. So the fact we're explicitly setting a `color` makes sense. But I don't think we should name it after one of the component library colours. It's really part of the dashboard theme IMO.

```suggestion
					style={ { color: 'var(--dashboard__divider-color)' } }
```

In `style.scss` you can set `--dashboard__divider-color: var(--wp-components-color-gray-100, #f0f0f0);` if you want. But it seems to me, like the background colour, the colour is fully decided by our app, and so therefore it should be a `--dashboard__` variable.